ops/storage-ddls/cassandra-setup.cql

```diff
@@ -208,3 +208,33 @@ WITH OPTIONS = {
 'tokenization_enable_stemming': 'true',
 'tokenization_normalize_lowercase': 'true'
 };
+
+DROP TABLE IF EXISTS timeseriestrends;
+CREATE TABLE timeseriestrends(
+    periodstartdate timestamp,
+    periodtype text,
+    period text,
+    tilez int,
+    tilex int,
+    tiley int,
+    trendscore float,
+    insertion_time timestamp,
+    topic text,
+    PRIMARY KEY ((periodtype, topic, tilez), periodstartdate, period, tilex, tiley)
```

Comment:
Could you provide some sample queries of how this data will be accessed in the UI? This would help me better understand the primary/clustering key choices.